### PR TITLE
Create ui-c8y-1020-13-0-has-external-origin-resolved-in-6724-graft-release-cd-6776.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60085](https://cumulocity.atlassian.net/browse/MTM-60085)
Original PR: [6776](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6776)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1020-13-0-has-external-origin-resolved-in-6724-graft-release-cd-6776.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: hasExternalOrigin resolved in (#6724) [GRAFT][release/cd] (#6776)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-13-0-has-external-origin-resolved-in-6724-graft-release-cd-6776.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Resolve hasExternalOrigin in release/cd branch
```

File: content/change-logs/platform-services/ui-c8y-1020-13-0-has-external-origin-resolved-in-6724-graft-release-cd-6776.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60085
+version: 1020.13.0
+---
+hasExternalOrigin resolved in (#6724) [GRAFT][release/cd] (#6776)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-13-0-has-external-origin-resolved-in-6724-graft-release-cd-6776.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the context of the release/cd branch, the hasExternalOrigin property was not being correctly resolved, causing issues with determining the origin of certain resources. This change fixes the resolution of hasExternalOrigin by properly handling the property in the relevant code paths. The impact of this fix is that the origin of resources will now be accurately determined in the release/cd branch, improving reliability and consistency.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by pink-mongoose — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1020-13-0-has-external-origin-resolved-in-6724-graft-release-cd-6776.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: The page with the application access for the user loads correctly.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-13-0-has-external-origin-resolved-in-6724-graft-release-cd-6776.md:3]:
```suggestion
title: Page with information on application access for the user loads correctly
```

File: content/change-logs/platform-services/ui-c8y-1020-13-0-has-external-origin-resolved-in-6724-graft-release-cd-6776.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60085
+version: 1020.13.0
+---
+During the loading of the page with application access settings for the user, an error occurred when verifying whether the user was created by an external authorization server. The error has been resolved.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-13-0-has-external-origin-resolved-in-6724-graft-release-cd-6776.md:17]:
```suggestion
During the loading of the page with the application access settings for the user, an error occurred when verifying whether the user was created by an external authorization server. The error has been resolved.
```

**Review by BeateRixen — APPROVED** [review]

**Review by kgol-c8y — APPROVED** [review]

**Pull request merged**